### PR TITLE
:sparkles: :egg: add male kelly's birthday

### PR DESCRIPTION
**Pull request by twentylemon**
##### Summary 

Title. Chnages were done on the website, so I haven't run tests. Letting the actions do that.

##### Checklist

* [x] this is a source code change
  * [ ] run `format` in the repository root
  * [ ] run `pytest` in the repository root
  * [x] link to issue being fixed using a [_fixes keyword_](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue), if such an issue exists
  * [x] update the wiki documentation if necessary
* [ ] or, this is **not** a source code change

### CODE REVIEW

File: tests/cogs/announce_day/announce_day_test.py [hunk 1]
```diff
@@ -31,10 +31,10 @@ async def test_cog_unload_cancels_task(bot, dog_photos):
 @mock.patch("random.choice", side_effect=["today", "tomorrow", "yesterday", "{today} {tomorrow} {yesterday}"])
 @mock.patch("random.random", return_value=0.5)
 async def test_on_hour_7am_eastern_special_day(random, choice, bot, dog_photos, general_channel):
-    with patch_now(datetime.datetime(2002, 1, 1, hour=7)):
+    with patch_now(datetime.datetime(2002, 2, 2, hour=7)):
         clazz = AnnounceDay(bot, dog_photos)
         await clazz.on_hour()
-        general_channel.send.assert_called_once_with("today tomorrow yesterday\nIt is also New Year's Day.")
+        general_channel.send.assert_called_once_with("today tomorrow yesterday\nIt is also Groundhog Day.")
```
Comment (jameshughes89) [inline tests/cogs/announce_day/announce_day_test.py:37]: but why

File: duckbot/cogs/announce_day/special_days.py [hunk 2]
```diff
@@ -22,6 +22,7 @@ def _populate(self, year):
 
         tito = discord.utils.get(self.bot.emojis, guild__name="Friends Chat", name="tito")
 
+        self[date(year, 1, 1)] = f"Male Kelly's birthday. He's {year-1900} years old according to most websites"
```
Comment (jameshughes89) [inline duckbot/cogs/announce_day/special_days.py:25]: nice

**Review by jameshughes89 — APPROVED** [review]

File: tests/cogs/announce_day/announce_day_test.py (hunk 1 above)
Comment (twentylemon) [reply tests/cogs/announce_day/announce_day_test.py:37]: Jan 1 is the birthday, so the message that got sent changes, so the test failed. Figured it'd be cleaner to use a shorter special day message.

Comment (jameshughes89) [reply tests/cogs/announce_day/announce_day_test.py:37]: Makes sense.

**Pull request merged**